content/english/python-tensorflow/activity-7.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Activity 7 - Plotting an Image's predicted Category"
+date: 2020-09-09T16:19:17-07:00
+weight: 12
+draft: false
+---
+
+<iframe width="100%" height="600px" src="https://www.youtube.com/embed/pwZDPj4yIsM" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+```python
```

Comment:
Add a comment stating to copy paste this code?

Reply:
Resolved